pom.xml

```diff
@@ -804,6 +804,79 @@
         <backend_type>velox</backend_type>
       </properties>
     </profile>
+    <profile>
+      <id>kafka</id>
+      <activation>
+        <activeByDefault>false</activeByDefault>
+      </activation>
```

Comment:
What's the consideration of making kafka support an optional feature of Gluten? Does it introduce a bunch more Jar dependencies that may cause unwanted dependency conflicts?

Reply:
Avoid unnecessary jar dependencies for structured streaming. It will be an experimental function of native streaming for a long time. It will not affect the current mainline function development.